proto/viam/app/data/v1/data.proto

```diff
@@ -374,18 +382,22 @@ message RemoveBoundingBoxFromImageByIDRequest {
 
 message RemoveBoundingBoxFromImageByIDResponse {}
 
+// BoundingBoxLabelsByFilterRequest requests all the labels of the bounding boxes from files from a given filter
 message BoundingBoxLabelsByFilterRequest {
   Filter filter = 1;
 }
 
+// BoundingBoxLabelsByFilterRequest returns all the labels of the bounding boxes from files from a given filter
 message BoundingBoxLabelsByFilterResponse {
   repeated string labels = 1;
 }
 
+// GetDatabaseConnectionRequest returns the database connection hostname endpoint for a given organization
```

Comment:
@maxhorowitz Could you look here and below to make sure the docs make sense?

Reply:
[nit] 'requests the database connection hostname'. The RPC already specifies that it's for ADF so agree that don't need to repeat here.